ApplicationLibCode/Application/RiaDefines.h

```diff
@@ -125,22 +125,26 @@ enum class PhaseType
 
 enum class ImportFileType
 {
-    NOT_A_VALID_IMPORT_FILE = 0x00,
```

Comment:
Defina a normal enum here, and remove the combination enums ECLIPSE_RESULT_GRID, ...

Reply:
Fixed in new attempt.